optuna/samplers/_cmaes.py

```diff
@@ -560,7 +560,12 @@ def _init_optimizer(
             for i, dist in enumerate(trans._search_space.values()):
                 assert isinstance(dist, (IntDistribution, FloatDistribution))
                 # Set step 0.0 for continuous search space.
-                steps[i] = dist.step or 0.0
+                if dist.step is None:
+                    steps[i] = 0.0
+                elif dist.low == dist.high:
+                    steps[i] = 1.0
+                else:
+                    steps[i] = dist.step / (dist.low - dist.high)
```

Comment:
When given `FloatDistribution(0, 10, step=2)`, the `steps[i]` should be `0.2`. However, in this code, it looks that `steps[i]` is set to `-0.2`. What do you think?

```suggestion
                    steps[i] = dist.step / (dist.high - dist.low)
```